frontend/src/App.jsx

```diff
@@ -21,14 +22,43 @@ function App() {
     console.log(response)
   }
 
+  const testDB = async () => {
+    const token = session.access_token;
+    const data = {
+      case_no: "1234", 
+      patient_id: "Jomari Francisco",
+      type: "men", 
+      surgeon: "Dr. Stove", 
+      or_date: "02/11/2024", 
+      age: "22", 
+      sex: "male", 
+      reason: "too cool for school", 
+      hpi: "high heartrate", 
+      social: "@jomarifrancisco",
+    }
+    try {
+      const response = await axios.post("http://localhost:8080/api/log/cardiacSurgeryAdultService", data, {
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Content-Type': 'application/json',
+        }
+      });
+
+      console.log(response);
+      setDatabaseTest(response.data.message);
+    } catch (err) {
```

Comment:
If there is an error, would recommend setting the appropriate states to display an error message directly on screen

Reply:
This should function already if there was an error inserting into database, but I will modify the catch block so it will display if the post request displays an error